### DISCUSSION TITLE
Poi

### DISCUSSION THREAD

**Discussion by anten1222** [osmandapp/OsmAnd#14519] [Ideas]
Custom Search
  For a normal or occasional user, this is suboptimal

  My suggestion below
  Show PIO OVERLAY
  "Other POI filters"

**Comment by scaidermern** [comment]
I don't get what you are suggesting. Can you rephrase your suggestion? Or maybe add an example?

**Reply by anten1222** [reply]
I mean it is now
 For a normal or occasional user, this is suboptimal
 Configure Map > Poi Overlay > Search
 Then at the very bottom Custom Search
 Only then you have access to all Pio

 my proposition
 Configure Map > Poi Overlay
 Then at the top Custom Search
